src/Toast/ToastContainer.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+class ToastContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.toastRootName = 'toast-root';
+    if (document.getElementById(this.toastRootName)) {
+      this.parentElement = document.getElementById(this.toastRootName);
```

Comment:
I would keep the naming of rootElement or parentElement consistent. (maybe stick with root?).